src/main/java/com/github/mauricioaniche/ck/util/StringUtils.java

```diff
@@ -15,4 +18,14 @@ public static int countLines(String str) {
         }
         return lines;
     }
-}
+
+    //Use the given regex pattern to extract a single substring from the given String
+    //Returns the empty string in case of no matching
+    public static String substringRegex(String string, String regex){
```

Comment:
this method would disappear if we use JDT's capabilities to get the variable's name

Reply:
Done